Khan/wonder-blocks
# Add Typography documentation to Storybook

## Summary:
Migrate the Typography documentation from styleguidist to Storybook.

(I used the same examples as the ones from styleguidist.)

Issue: https://khanacademy.atlassian.net/browse/WB-1147

## Test plan:
`yarn flow`
`yarn run lint`

Storybook
- Alter values in the control panel and confirm that the example changes accordingly.
  - Add {"color": "blue"} to style prop and confirm the text becomes blue.
- Go to "Noto for non-Latin languages" story and change the values in the
  dropdown. Confirm it changes the languages correctly.
